### DISCUSSION TITLE
Possibility to input own tilemap for spatial/gis viewer

### DISCUSSION THREAD

**Discussion by 0red** [dbeaver/dbeaver#15258] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Actually only 2 background map are supported.
The feature is to add possibility to input the additional tilemap server url for presenting the results on the other map layer

**Describe the solution you'd like**
Preferences,Editors,Data Editor,GIS Viever add the text indut field to provide by the user the url for tile server (i.e. in the [leafletjs.com](https://leafletjs.com/) ) format like https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png  (details [https://leafletjs.com/reference.html#tilelayer](https://leafletjs.com/reference.html#tilelayer) )

Then the new map alternative will be visible in the Spatial Result view

**Comment by ShadelessFox** [comment, marked as answer]
Hello @0red,

We already support that feature. Perhaps you're running on a really outdated version?

Please see https://github.com/dbeaver/dbeaver/wiki/Working-with-Spatial-GIS-data#defining-custom-tile-layer

**Comment by 0red** [comment]
so I missed it. Thx
